### ISSUE TITLE
Homebrew deprecation warning: bottle :unneeded

### ISSUE THREAD

**Issue by kevinkirkup** [jmespath/homebrew-jmespath#2]
```
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the jmespath/jmespath tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/jmespath/homebrew-jmespath/Formula/jp.rb:8
```

**Comment by b-beijer** [comment]
Hi, encounterd the same problem.
So for now, i  cloned the 'homebrew-jmespath` repo with the brew formulae and remove the offending statement.

Now you can use (remove jmespath/jmespath/jp first):

- `brew install  b-beijer/jmespath-mcc/jp`

And the brew warning disappears

Grtx,
Ben

PS See for yourself:
[homebrew formulae](https://github.com/b-beijer/homebrew-jmespath-mcc)

**Comment by nitrocode** [comment]
Thanks @b-beijer for the workaround. It would be nice to have it as an official fix so I can stop seeing this homebrew warning 🙏 

cc: @jamesls @skmcgrail if you have time, please consider the solution of removing the `bottle :unneeded` statement since it's no longer needed

**Comment by jamesls** [comment]
Fixed via #1
